go.mod

```diff
@@ -13,5 +14,6 @@ require (
 
 replace (
 	github.com/ethereum/go-ethereum => github.com/MariusVanDerWijden/go-ethereum v1.8.22-0.20211208130742-dd90624af970
+	github.com/ferranbt/fastssz => github.com/kasey/fastssz v0.0.0-20211117233602-01b170cf6dcc
```

Comment:
The latest `prysmaticlabs/fssz` is better. The latest fixes and upcoming optimizations are happening there:
https://github.com/prysmaticlabs/fastssz